editor.planx.uk/src/pages/PlatformAdminPanel.tsx

```diff
@@ -0,0 +1,167 @@
+  padding: theme.spacing(1),
+  "&::after": {
+    position: "absolute",
+    width: "100%",
+  },
+}));
+
+function Component() {
+  const adminPanelData = useStore((state) => state.adminPanelData);
+
+  return (
+    <Box p={3}>
+      <Typography variant="h1">Platform Admin Panel</Typography>
+      <Typography variant="body1" mb={3}>
+        {`This is an overview of each team's integrations and settings for the `}
+        <strong>{process.env.NODE_ENV}</strong>
+        {` environment`}
+      </Typography>
+      {adminPanelData?.map((team) => <TeamData key={team.id} data={team} />)}
+    </Box>
+  );
+}
+
+interface TeamData {
+  data: AdminPanelData;
+}
+
+const NotConfigured: React.FC = () => <Close color="error" fontSize="small" />;
+
+const Configured: React.FC = () => <Done color="success" fontSize="small" />;
+
+const TeamData: React.FC<TeamData> = ({ data }) => {
+  return (
+    <StyledTeamAccordion primaryColour={data.primaryColour}>
+      <AccordionSummary
+        id={`${data.name}-header`}
+        aria-controls={`${data.name}-panel`}
+        expandIcon={<Caret />}
+        sx={{ pr: 1.5 }}
+      >
+        <Typography variant="h2">{data.name}</Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Grid
+          container
+          direction="row"
+          justifyContent="flex-start"
+          alignItems="flex-start"
+          spacing={3}
+        >
+          <Grid item xs={4}>
+            <SummaryListTable dense={true}>
+              <>
+                <Box component="dt">{"Slug"}</Box>
+                <Box component="dd">
+                  <code>
+                    {`/`}
+                    {data.slug}
+                  </code>
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Homepage"}</Box>
+                <Box component="dd">{data.homepage || <NotConfigured />}</Box>
+              </>
+              <>
+                <Box component="dt">{"Logo"}</Box>
+                <Box component="dd">
+                  {data.logo ? <Configured /> : <NotConfigured />}
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Favicon"}</Box>
+                <Box component="dd">
+                  {data.favicon ? <Configured /> : <NotConfigured />}
+                </Box>
+              </>
+            </SummaryListTable>
+          </Grid>
+          <Grid item xs={4}>
+            <SummaryListTable dense={true}>
+              <>
+                <Box component="dt">{"Planning constraints"}</Box>
+                <Box component="dd">
+                  {data.planningDataEnabled ? (
+                    <Configured />
+                  ) : (
+                    <NotConfigured />
+                  )}
+                </Box>
+              </>
+              <>
+                <Box component="dt">{"Article 4s"}</Box>
+                <Box component="dd">{"?"}</Box>
```

Comment:
Followup PR: figure out how to check if granular A4s are being returning by our Planx GIS API yet ! 

This is perhaps the trickiest one to currently measure (do we mean the spreadsheet, flow, API response etc) - but I've included this "?" placeholder as a reminder to manually check.

Reply:
Yep I was thinking of a similar endpoint variation - agree API is the thing we care about checking most here compared to flow content.

Will plan to pick this one back up when I pick up other constraints work in the coming weeks!